stories/why-input.stories.jsx

```diff
@@ -0,0 +1,65 @@
+import { userEvent, within } from "@storybook/testing-library";
+import React from 'react';
+import WhyInput from "../app/components/why-input";
+import expect from 'expect';
+import { onDoneDecorator, onDoneResult } from "./common";
+import DemInfo from '../app/components/dem-info';
+
+export default {
+    component: WhyInput,
+    args: {},
+    decorators: [onDoneDecorator],
+}
+
+const user = {
+    dob: '1990-10-20T00:00:00.000Z',
+    state: 'NY',
+    party: 'Independent'
+}
+
+export const ExamplePoint = {
+    args: {
+        point: {
+            subject: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at bibendum sapien",
+            description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at bibendum sapien",
+            vState: "secondary",
```

Comment:
vState shouldn't be part of the data, it should be part of the code.  As I understand it here, the WhyInput component always wants to pass the secondary vState to the point so it will have this look, so it should be hard a hard coded prop that is passed.